cirq-core/cirq/ops/projector.py

```diff
@@ -61,7 +61,7 @@ def matrix(self, projector_qids: Optional[Iterable[raw_types.Qid]] = None) -> cs
             for qid in projector_qids
         ]
 
-        total_d = np.prod([qid.dimension for qid in projector_qids], dtype=np.int64)
+        total_d = np.prod([qid.dimension for qid in projector_qids], dtype=np.int64).item()
```

Comment:
consider using [`math.prod`](https://docs.python.org/3/library/math.html#math.prod) instead
```suggestion
        total_d = math.prod(qid.dimension for qid in projector_qids)
```

Reply:
Nice!  Done in b32f04e56eb82b01741523a9fff2fb893c6257cc